docs/update.md

```diff
@@ -182,9 +182,12 @@ Manual update of Percona XtraDB Cluster can be done as follows:
 
     !!! warning
 
-        The above command upgrades various components of the cluster including PMM Client. It is [highly recommended](https://docs.percona.com/percona-monitoring-and-management/how-to/upgrade.html) to upgrade PMM Server **before** upgrading PMM Client. If it wasn't done and you would like to avoid PMM Client upgrade, remove it from the list of images, reducing the last of two patch commands as follows:
-        
-        
+        The above command upgrades various components of the cluster including PMM Client and HA Proxy, which may need additional attention.
+
+        * If you are using [custom configuration for HA Proxy](haproxy-conf.md#passing-custom-configuration-options-to-haproxy), check [HA Proxy release information](https://www.haproxy.org/) to make sure that the version you are applying is compatible with your existing custom config.
```

Comment:
@fiowro, the issue is that we occasionally add new options to the HAProxy configuration. If users are using a custom configuration for HAProxy, they need to check the default configuration at https://github.com/percona/percona-docker/blob/pxc-operator-1.14.0/haproxy/dockerdir/etc/haproxy/haproxy-global.cfg and incorporate any new options into their custom config.

For example, in this commit: https://github.com/percona/percona-docker/commit/f2b9f63e8c288d219522ea76ae6b113e2717a59c, we added two new options in different places. If these options are not included in the custom config, users may experience downtime.

The instructions are as follows:
1. check new conf before the update
2. add new options while the update into CR